src/app/models/propTypes/inline/index.js

```diff
@@ -1,13 +1,14 @@
-import { string, arrayOf, oneOf, shape } from 'prop-types';
+import { string } from 'prop-types';
+import { blockOfTypesAndModel, blocksWithTypes } from '../general';
 import fragmentBlockPropTypes from '../fragment';
 
-const inlineBlockPropTypes = shape({
-  type: 'inline',
-  model: shape({
-    text: string.isRequired,
-    blocks: arrayOf(oneOf([fragmentBlockPropTypes])),
-    language: string.isRequired,
-  }),
-});
+export const inlineModelPropTypes = {
+  language: string.isRequired,
+  text: string.isRequired,
+  ...blocksWithTypes([fragmentBlockPropTypes]),
+};
 
-export default inlineBlockPropTypes;
+export const inlineBlockPropTypes = blockOfTypesAndModel(
+  ['inline'],
+  inlineModelPropTypes,
```

Comment:
```suggestion
  model: inlineModelPropTypes,
```
is this not a model?

Reply:
This is already handled by https://github.com/bbc/simorgh/blob/9539b4a0ab5f83547c1a5612534b776c3d678152/src/app/models/propTypes/general/index.js#L6